docs/components/gantry/multi-axis.md

```diff
@@ -211,3 +211,5 @@ The following attributes are available for `multi-axis` gantries:
 | --------- | ---- | --------- | ----------- |
 | `subaxes_list` | array | **Required** | An array of the `name` of each of the sub-axes, the [`single-axis`](/components/gantry/single-axis/) gantries that make up the `multi-axis` gantry. |
 | `move_simultaneously` | boolean | Optional | A boolean indicating if the sub-axes should move together, or one at a time when `MoveToPosition` is called. <br> Default:  `false` |
+
+If you are using the `limit_pins` attribute with one or more of your `single-axis` gantries, and you require that homing routines for each `single-axis` gantry take place in a certain order, you can [specify the order of homing routines](/components/gantry/single-axis/#specify-the-order-of-homing-routines) if needed.
```

Comment:
no longer relevant because there is no more automatic homing so homing can be called in whatever order the user needs